src/partials/nav.html

```diff
@@ -168,6 +168,14 @@
             </li>
           </ul>
         </li>
+        <li class="nav-item">
+          <div
+            class="nav-link"
+            id="localeSelector"
+            role="button"
```

Comment:
Add `role` with your JS; if JS fails to load, this will be a broken button for users with screen readers.

Reply:
Added

Reply:
Since we're adding this with JS, let's remove it from the HTML source:

```suggestion
```